src/test/scala/ru/d4team/chat/dummy/services/DummySpec.scala

```diff
@@ -0,0 +1,39 @@
+package ru.d4team.chat.dummy.services
+
+import ru.d4team.chat.dummy.Generators.genDummy
+import ru.d4team.chat.dummy.models.{Dummy, DummyResult}
+import zio._
+import zio.mock.Expectation.value
+import zio.mock.mockable
+import zio.test.Assertion.{anything, equalTo}
+import zio.test._
+
+object DummySpec extends ZIOSpecDefault {
+
+  @mockable[ChildDummyServiceA]
+  object MockDummyA
+
+  @mockable[ChildDummyServiceB]
+  object MockDummyB
+
+  override def spec: Spec[TestEnvironment with Scope, Any] = suite("DummySpec")(
+    test("should show how tests work") {
+      check(genDummy) { dummy =>
+        val changeResult = Dummy("whoKnows?")
+        val expected     = DummyResult(42)
+
+        // WHEN
```

Comment:
The common test structure usually describes via Given-When-Then (GWT) structured.
So, it's better to use these words for comments:
```scala
// given
// when
// then
```

Reply:
done)